examples/dmd/generic_csv.cpp

```diff
@@ -20,7 +20,6 @@
 // 5. DATA_DIR/DATASET/STATE/tval.csv     -- specifies the time instance of STATE
 // 6. DATA_DIR/dim.csv                    -- specifies the dimension of VAR_NAME
 // 7. DATA_DIR/index.csv                  -- (optional) each row specifies one DOF of VAR_NAME
-// 8. DATA_DIR/indicator_val.csv          -- (optional) each row specifies one indicator endpoint value
```

Comment:
It would be helpful to have sample runs, as in `heat_conduction.cpp`.

Reply:
For the purpose of adding sample runs, I have run the parametric heat conduction MFEM example and stored the outputs as CSV, so that it can be fed to the `generic_csv`. I planned to add the CSV files in `example/dmd`, but it is 700KB. I guess it is too large to be directly included into the repo?

Reply:
Perhaps, you can include generic_csv for a small number of time steps only.

Reply:
Following the MFEM parametric heat conduction problem, 5 datasets, each with 4 time steps, are added.